deform/widget.py

```diff
@@ -141,6 +141,12 @@ class Widget(object):
         be added to the input tag.
 
     requirements
+
+        The requirements are specified as a a sequence of either:
+          1. two-tuples in the form ``(requirement_name, version_id)``. Using the
+            resource registry the **logical** requirement name identifiers are resolved
+            to concrete files using the resource_registry
+          2. dicts that contcretly point to resources, e.g. ``{"js": "deform:static/tinymce/tinymce.min.js"}``
         A sequence of two-tuples in the form ``( (requirement_name,
```

Comment:
The above needs to be formatted for reST and Black for linting. Leading and trailing blank lines are needed for reST lists. For Black line wraps must be less than 79 characters in docstrings. In narrative text, we don't use line wraps.

```

            1.  two-tuples in the form ``(requirement_name, version_id)``.
                Using the resource registry the **logical** requirement name
                identifiers are resolved to concrete files using the
                ``resource_registry``, or
            2.  dicts that concretely point to resources such as ``{"js":
                "deform:static/tinymce/tinymce.min.js"}``.

```

Reply:
thanks, I never write reST